### DISCUSSION TITLE
Colour replacement

### DISCUSSION THREAD

**Discussion by stack-bblincoe** [svrnm/DemoMonkey#135] [Q&A]
Hi, 

When I've used DM previously it was possible to replace colours within the site. 

Is this functionality still available and if so what is the syntax?

**Comment by svrnm** [comment]
Hey :-) 

it depends a little bit what you want to recolor, there are some AppD specific commands, but there are also some generic options like

!recolorImage(image) = rgb(1,2,3)
!style(Replacement, color) = red (and you can use background-color or other css attributes for that)
!querySelector(cssSelector, style.background) = blue

Note that there are many circumstances where this might not work, so if you can share an example with me what you'd like to accomplish then I can give concrete pointers.

Cheers,
Severin

**Comment by stack-bblincoe** [comment]
Thanks for taking a look Severin, Really appreciate it.

I'm hoping to replace the colour of the top bar on this site:
https://soedemo.stackenterprise.co/

The login page seems to have the same structure as the main site, so
hopefully this is enough to put us in the right direction.

Regards

Ben

**Comment by svrnm** [comment]
Can you try the following:

```
!querySelector(.s-topbar, style.background-color) = rgb(0,255,0) 
```

**Comment by stack-bblincoe** [comment]
Worked perfectly!

Thank you.



Kind Regards,

Ben Blincoe (He/Him)

Solutions Engineering Manager - EMEA/ROW
+447779337451
